CoreVulkanDisplay/src/au/gov/asd/tac/constellation/visual/vulkan/shaders/PassThruPoint.vs

```diff
@@ -12,5 +12,8 @@ layout(location = 0) in vec3 vertex;
 
 void main(void) {
     gl_PointSize = 4.0;
-    gl_Position = ub.mvpMatrix * vec4(vertex.x, -vertex.y, vertex.z, 1);
+
+    gl_Position = ub.mvpMatrix * vec4(vertex.x, vertex.y, vertex.z, 1);
+    gl_Position.y = -gl_Position.y;
```

Comment:
Does the Y conversion have to be in the shader? I thought it was translated in the matrices/points before it hits the shader pipeline?

Reply:
Yes it does as this is where we do the screen space transform.  Just flipping a mtx(1,1) value gives the wrong result as the view portion of our matrix has a translation component (because the camera moves).